openassessment/xblock/test/test_rubric_reuse_mixin.py

```diff
@@ -0,0 +1,162 @@
+        """ Test that list_all_other_ora_block_locators does not include orphaned ORA blocks """
+        non_orphans = [
+            self._make_mock_ora_block() for _ in range(3)
+        ]
+        orphans = [
+            self._make_mock_ora_block(orphan=True) for _ in range(2)
+        ]
+
+        course_ora_blocks = [self.block] + non_orphans + orphans
+        self.mock_get_course_ora_blocks(return_value=course_ora_blocks)
+
+        result = self.block.get_other_course_ora_blocks()
+        self.assertEqual(set(result), set(non_orphans))
+
+    # -----------
+    # get_rubric
+    # -----------
+
+    def _request_get_rubric(self, payload):
+        """ Helper function for calling the XBlock JSON handler """
+        return self.request(
+            self.block,
+            'get_rubric',
+            json.dumps(payload),
+            response_format='json'
+        )
+
+    def test_get_rubric(self):
+        """ Test that get_rubric will get a rubric correctly"""
+        another_ora = self._make_mock_ora_block(location=self.other_block_location)
+        another_ora.rubric_criteria = "this is an arbitrary value for the rubric"
+        another_ora.rubric_feedback_prompt = "this is an arbitrary value for the feedback prompt"
+        another_ora.rubric_feedback_default_text = "this is an arbitrary value for the default feedback text"
+
+        with self.mock_get_ora_block(return_value=another_ora):
+            resp = self._request_get_rubric({'target_rubric_block_id': str(self.other_block_location)})
+        self.assertTrue(resp['success'])
+        self.assertEqual(resp['rubric']['criteria'], another_ora.rubric_criteria)
+        self.assertEqual(resp['rubric']['feedback_prompt'], another_ora.rubric_feedback_prompt)
+        self.assertEqual(resp['rubric']['feedback_default_text'], another_ora.rubric_feedback_default_text)
+
+    def test_get_rubric__missing_param(self):
+        """ Test for get_rubric error behavior when called with missing params """
+        resp = self._request_get_rubric({})
+        self.assertFalse(resp['success'])
+        self.assertEqual(resp['msg'], 'You must specify a block id from which to copy a rubric.')
+
+    def test_get_rubric__invalid_key(self):
+        """ Test for get_rubric error behavior when the parameter is an invalid key """
+        resp = self._request_get_rubric({'target_rubric_block_id': 'invalidKEEEEY'})
+        self.assertFalse(resp['success'])
+        self.assertEqual(resp['msg'], 'Invalid block id.')
+
+    def test_get_rubric__not_found(self):
+        """ Test for get_rubric error behavior when the requested block can't be found """
+        with self.mock_get_ora_block(side_effect=TargetORABlockNotFoundException):
+            resp = self._request_get_rubric({'target_rubric_block_id': self.OTHER_BLOCK_LOCATION})
+        self.assertDictEqual(
+            resp,
+            self.block._get_item_error_response(self.other_block_location)
```

Comment:
`_get_item_error_response` was not mock or implemented. Was it support to be inherit from somewhere else?